src/components/MediaSettingsModal/MediaSettingsSchema.jsx

```diff
@@ -27,6 +27,13 @@ export const MediaSettingsSchema = (existingTitlesArray = []) =>
           return (value.match(/\./g) || []).length <= 1
         }
       )
+      .test(
+        "File not supported",
+        "File names cannot begin with an underscore",
+        (value) => {
+          return !value.startsWith("_")
+        }
+      )
```

Comment:
Hmm this is only a check for the user-provided name yeah? On the backend I believe we convert special characters into `_`, so something like `!!image.jpg` will be converted to `__image.jpg` and bypass this check. I think we might either need to tighten up this check to block starting with special characters entirely, or move this check to the backend?

Reply:
this is a frontend thing. keeping the rules consistent for both image upload and renames